### ISSUE TITLE
Meeting #3

### ISSUE THREAD

**Issue by Trott** [nodejs/testing#17]
@nodejs/testing Sorry for the late notice. Will this still work for enough people to make the meeting worthwhile? 

UTC: Friday, February 26, 2016 at 6:00:00 PM
Pacific Time (San Francisco): Friday, February 26, 2016 at 10:00:00 AM
Eastern Time (Pittsburgh): Friday, February 26, 2016 at 1:00:00 PM
Eastern Caribbean Time (Barbados): Frida, February 26, 2016 at 2:00:00 PM
Central European Time (Madrid): Friday, February 26, 2016 at 7:00:00 PM
Western European Time (Lisbon): Friday, February 26, 2016 at 6:00:00 PM

Agenda carry-overs from last time:
- How to motivate people to fix tests
- Revise Flaky Test wiki page (Rich has a deliverable on this one)
- Skipped Test Catalog

Any other items? Please comment in this issue or open a separate issue and tag `wg-agenda`.

**Comment by cjihrig** [comment]
Works for me.

**Comment by MylesBorins** [comment]
works for me

**Comment by joaocgreis** [comment]
Ok for me.

**Comment by orangemocha** [comment]
:+1:

**Comment by santigimeno** [comment]
I can make it

**Comment by Trott** [comment]
Hangout link: https://plus.google.com/hangouts/_/hoaevent/AP36tYcSAlhAVME050Q1Hb8fWiEw56oEyvb-lvrDwvU_5OMDUuzB2A?hl=en&authuser=0

**Comment by Trott** [comment]
Google Doc for minutes:
https://docs.google.com/document/d/1Xc4yWmskxuAdxpRgHw7XzFcOM-AsFVrBOntN8VC_DOc/edit?usp=sharing

**Comment by Trott** [comment]
Updated agenda:
- How to motivate people to fix tests (Alexis)
- Revise Flaky Test wiki page (Rich/Alexis) https://github.com/nodejs/testing/issues/13
- Skipped Test Catalog (Alexis?)
- Single stress testing of parallel tests in parallel https://github.com/nodejs/testing/issues/3
- CI results searching via automation or... https://github.com/nodejs/testing/issues/7
- Flaky tests https://github.com/nodejs/testing/issues/11

**Comment by Trott** [comment]
Minutes: https://docs.google.com/document/d/1Xc4yWmskxuAdxpRgHw7XzFcOM-AsFVrBOntN8VC_DOc/edit

Please review/comment/edit if you see anything missing or wrong.

**Comment by Trott** [comment]
Recording: https://www.youtube.com/watch?v=7rL59c8a5Qk
